tests/test_read_write/test_read_jsonl.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import os
+from clumper import Clumper
+
+
+@pytest.mark.datafiles("./tests/test_read_write/sample_jsonl_files/cards.jsonl")
+def test_read_jsonl_all_lines(datafiles):
+    path = str(datafiles)  # Convert from py.path object to path (str)
+
+    file_path = str(os.path.join(path, "cards.jsonl"))
+    assert os.path.isfile(file_path)  # Make sure its a file
+    assert (
+        len(Clumper.read_jsonl(file_path)) == 4
+    )  # There are 4 rows in the sample file
+
+
+@pytest.mark.datafiles("./tests/test_read_write/sample_jsonl_files/cards.jsonl")
```

Comment:
Might be nice to add a `pytest.mark.parametrize` here so we can check that it works for 0 lines, 1 line, 2 lines, etc.

Reply:
Good idea!